packages/arb-token-bridge-ui/src/components/common/AddCustomChain.tsx

```diff
@@ -285,29 +285,29 @@ export const AddCustomChain = () => {
       <textarea
         onChange={e => setChainJson(e.target.value)}
         placeholder="Insert the JSON configuration from the `outputInfo.json` file that's generated at the end of the custom Orbit chain deployment."
-        className="min-h-[100px] w-full rounded-lg px-4 py-2 text-sm font-light text-black"
+        className="min-h-[154px] w-full rounded border border-gray-dark bg-dark p-4 text-sm font-light text-white placeholder:text-gray-3"
       />
       {error && (
         <div className="relative">
-          <pre className="scroll mb-2 max-h-[400px] overflow-auto rounded-lg border border-white/20 bg-white/5 p-4 text-sm text-error">
+          <pre className="scroll mb-2 max-h-[400px] overflow-auto rounded border border-gray-dark bg-dark p-4 text-sm text-error">
             <button
               onClick={() => setError(null)}
               className="arb-hover absolute right-4 top-4 text-white"
             >
-              <XMarkIcon width={24} />
+              <XMarkIcon width={20} />
             </button>
             {error}
           </pre>
         </div>
       )}
       <div className="flex w-full justify-end">
         {addingChain ? (
-          <Loader size="small" />
+          <Loader size="small" color="white" />
         ) : (
           // Need to replace with an atom
           <button
             onClick={onAddChain}
-            className="rounded bg-white p-2 text-sm text-black transition-all hover:opacity-80 disabled:cursor-not-allowed disabled:opacity-50"
+            className="arb-hover rounded bg-white p-2 text-sm text-black transition-all disabled:cursor-not-allowed disabled:opacity-50"
```

Comment:
check disabled state styles on Figma

disabled state text colour should be `var(--Gray-4, #CCC);`